README.md

```diff
@@ -2,9 +2,47 @@
+| [State of CSS](https://stateofcss.com/)                                     | Devographics            | Jun 2023 | 9108    | N/A                                                             | [Results](https://2023.stateofcss.com/en-US/)                                                              |
+| [Web Security](mdn-short-surveys/2023-05-15-security-dx)                    | W3C workshop / MDN      | May 2023 | 297     | [CSV](mdn-short-surveys/results.csv)                            | [Interpretation](mdn-short-surveys/2023-05-15-security-dx/interpretation.md)                               |
+| [Web Components](mdn-short-surveys/2023-03-31-web-components)               | Web Components CG / MDN | Mar 2023 | 732     | [CSV](mdn-short-surveys/2023-03-31-web-components/results.csv)  | [Interpretation](mdn-short-surveys/2023-03-31-web-components/interpretation.md)                            |
+| [DevSat Survey](vendor-research/browser-support-matrix-survey.pdf)          | Google                  | Mar 2023 | 852     | N/A                                                             | [PDF slides](vendor-research/Web%20Dev%20Sat%20H1%202023.pdf)                                              |
+| [Browser Support Matrix](vendor-research/browser-support-matrix-survey.pdf) | Google                  | Nov 2022 | 233     | N/A                                                             | [PDF slides](vendor-research/browser-support-matrix-survey.pdf)                                            |
+| [Web Browsers](mdn-short-surveys/2022-10-24-browser-support)                | WebDX / MDN             | Oct 2022 | 964     | [CSV](mdn-short-surveys/2022-10-24-browser-support/results.csv) | [Interpretation](mdn-short-surveys/2022-10-24-browser-support/interpretation.md)                           |
+                                           |
+
+
+## Milestones
+
+An explicit goal of some of the recurring surveys is to inform [the Interop project](https://github.com/web-platform-tests/interop) on priorities. These surveys need to run in advance of the project's selection process, usually **in October**.
+
+More broadly speaking, important milestones for recurring surveys tracked by the WebDX Community Group are:
+
+| Survey                                        | Start | Review | Date | Results |
+|-----------------------------------------------|-------|--------|------|---------|
+| [State of HTML](https://stateofhtml.com/)     | Mar   | Jun    | Jul  | Oct     |
+| [State of CSS](https://stateofcss.com/)       | Feb   | May    | Jun  | Sep     |
+| [State of JavaScript](https://stateofjs.com/) | Aug   | Oct    | Nov  | Jan     |
+| DevSat survey H1                              | Dec   | Feb    | Mar  | Apr     |
+| DevSat survey H2                              | Jun   | Aug    | Sep  | Oct     |
```

Comment:
@atopal I made the months up for the most part ;) You might have a better perspective on dates, at least for 2024. Happy to adjust as needed!